src/App.scss

```
@@ -0,0 +1,96 @@
+body {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: Ubuntu, serif;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.app {
+  margin: 10vh auto;
+  width: 50vw;
+  height: 500px;
+  background: grey;
+}
+
+.homePage {
```

не пишем камелкейсом
```suggestion
.home_page {
```

исправь везде